packages/shared/lib/services/user.service.ts

```diff
@@ -66,10 +90,19 @@ class UserService {
         return result[0];
     }
 
-    async createUser(email: string, name: string, hashedPassword: string, salt: string, accountId: number): Promise<User | null> {
-        const result: Pick<User, 'id'> = await db.knex
-            .from<User>(`_nango_users`)
-            .insert({ email: email, name: name, hashed_password: hashedPassword, salt: salt, account_id: accountId }, ['id']);
+    async createUser(email: string, name: string, hashedPassword: string, salt: string, accountId: number, email_verified = true): Promise<User | null> {
+        const result: Pick<User, 'id'> = await db.knex.from<User>(`_nango_users`).insert(
+            {
+                email,
+                name,
+                hashed_password: hashedPassword,
+                salt: salt,
+                account_id: accountId,
+                email_verified,
+                email_verification_token: email_verified ? null : uuid.v4()
+            },
+            ['id']
```

Comment:
nitpick: I personally prefer using an explicit `returning()` that using the unnamed second parameter of `insert`.

Reply:
Updated